ideascube/monitoring/forms.py

```diff
@@ -46,18 +46,19 @@ def clean_serials(self):
         return set([s for s in serials.splitlines() if s])
 
 
-class ExportEntryForm(forms.Form):
+class SinceForm(forms.Form):
 
-    since = forms.DateField(
-        widget=forms.DateInput(format='%Y-%m-%d'),
-        required=False)
+    since = forms.DateField(label=_('since'),
+                            widget=forms.DateInput(format='%Y-%m-%d'),
+                            required=False)
```

Comment:
The way you've reindented everything after adding the `label=` argument makes this change bigger than it actually is, since we now have to review lines of code that haven't changed.

Compare with how much easier this is to review:

```diff
     since = forms.DateField(
+        label=_('since'),
         widget=forms.DateInput(format='%Y-%m-%d'),
         required=False)
```

Or even this, if you don't want to have too many lines:

```diff
     since = forms.DateField(
         widget=forms.DateInput(format='%Y-%m-%d'),
-        required=False)
+        required=False, label=_('since'))
```

And since the indentation was perfectly pep8-compliant before, I would really prefer if you had kept the change as small as possible, making it as easy to review as possible.